app/docker/views/services/servicesController.js

```diff
@@ -6,7 +6,7 @@ function ($q, $scope, $state, Service, ServiceService, ServiceHelper, Notificati
     publicURL: EndpointProvider.endpointPublicURL()
   };
 
-  $scope.scaleAction = function scaleService(service) {
+  $scope.scaleAction =  function scaleService(service) {
```

Comment:
Please remove the extra space here between the `=` char and `function`.

Reply:
no prob..will fix it and submit again.